app/workers/crawler/my_basket.rb

```diff
@@ -15,14 +15,18 @@ def fetch_places
     shop_names = page.xpath('//tr[@class="vcard"]/td[@class="fn org"]').map{|node| node.text }
     postal_codes = page.xpath('//tr[@class="vcard"]/td[@class="postal-code"]').map{|node| node.text }
     prefectures = page.xpath('//tr[@class="vcard"]/td[@class="adr"]').map { |node| node.xpath('./span[@class="region"]').text.squish.match(/東京都|神奈川県/).to_s }
-    addresses = page.xpath('//tr[@class="vcard"]/td[@class="adr"]').map { |node| node.xpath('./span[@class="locality"]').text + node.xpath('./span[@class="street-address"]').text }
+    addresses = page.xpath('//tr[@class="vcard"]/td[@class="adr"]').map { |node| node.xpath('./span[@class="locality"]').text + node.xpath('./span[@class="street-address"]').text.squish }
```

Comment:
__[rubocop]__
* Line is too long. [187/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)